JPS_Ontology/ontology/ontobuiltenv/OntoBuildingEnvironment.csv

```diff
@@ -83,7 +83,7 @@ Non-Domestic,Class,PropertyUsage,IS-A,,,,"A property that serves non-living purp
 SingleResidential,Class,Domestic,IS-A,,,,A property that serves as living space for one party.,https://www.theworldavatar.com/kg/ontobuiltenv/,Single residential
 MultiResidential,Class,Domestic,IS-A,,,,A property that serves as living space for more than one party.,https://www.theworldavatar.com/kg/ontobuiltenv/,Multi residential
 EmergencyService,Class,Non-Domestic,IS-A,,,,Emergency services are organizations which ensure public safety and health by addressing different emergencies.,https://www.theworldavatar.com/kg/ontobuiltenv/,Emergency service
-MedicalCare,Class,Non-Domestic,IS-A,,,,Medical care includes sites which focus on the provision of secondary medical care services.,https://www.theworldavatar.com/kg/ontobuiltenv/,Medical care
+Medical,Class,Non-Domestic,IS-A,,,,"Medical include buildings and sites focused on the provision of secondary medical care services or the dispensing of medicinal drugs, or both.",https://www.theworldavatar.com/kg/ontobuiltenv/,Medical
```

Comment:
I believe the trailing `"` are unnecessary and should be removed. Furthermore, we should just remove the "secondary", as this can be misleading: "Provision of medical care services..." is broader and describes better what we actually want to capture I believe.